spring-integration-core/src/test/kotlin/org/springframework/integration/dsl/KotlinDslTests.kt

```diff
@@ -345,8 +367,6 @@ class KotlinDslTests : TestApplicationContextAware {
 				}
 				transform<String> { it.uppercase() }
 			}
-
-
```

Comment:
Sorry to bother you too much for this comment, but it must not stick to the previous code.